__tests__/app.test.js

```diff
@@ -108,3 +108,57 @@ describe('"/api/reviews/:reviews_id', () => {
       });
   });
 });
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("200: responds with array of comments related to given review_id sorted by most recent first", () => {
+    return request(app)
+      .get("/api/reviews/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeInstanceOf(Array);

```

Comment:
We wouldn't need line 118 here as we're using forEach later, so know this is an array 👍